doc/conf.py

```diff
@@ -13,3 +13,7 @@
         project_name="ts_MTMount", version=lsst.ts.MTMount.__version__
     )
 )
+
+intersphinx_mapping["ts_xml"] = ("https://ts-xml.lsst.io", None)  # noqa
+intersphinx_mapping["ts_salobj"] = ("https://ts-salobj.lsst.io", None)  # noqa
+intersphinx_mapping["ts_hexrotcomm"] = ("https://ts-salobj.lsst.io", None)  # noqa
```

Comment:
Nice, I didn't know you could do that.

Reply:
I think Eric taught me that trick. It's really nice and avoids an annoying sphinx warning with ts_salobj v6 (which I could have worked around in salobj, but since this works and produces better docs I decided to live with it).